modules/file-integrity-operator-installing-cli.adoc

```diff
@@ -70,7 +63,7 @@ metadata:
   name: file-integrity-operator
   namespace: openshift-file-integrity
 spec:
-  channel: "${OC_VERSION}"
+  channel: "release-0.1"
```

Comment:
I am not sure do we need to point out it only suitable when version is equal or higher than file-integrity-operator.v0.1.16.

Reply:
We want everyone to install the latest, so I think this is OK.

Reply:
Okay.